llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -1419,6 +1425,12 @@ AArch64TargetLowering::AArch64TargetLowering(const TargetMachine &TM,
         setOperationAction(ISD::OR, VT, Custom);
     }
 
+    // LRINT and LLRINT.
+    for (auto VT : MVT::fp_scalable_vector_valuetypes()) {
```

Comment:
I think this would fit in the loop starting at line 1516
`for (auto VT : {MVT::nxv2f16, MVT::nxv4f16, MVT::nxv8f16, MVT::nxv2f32,...`